src/command_modules/azure-cli-network/azure/cli/command_modules/network/tests/test_network_commands.py

```diff
@@ -1543,7 +1543,7 @@ def test_network_subnet_endpoint_service(self, resource_group):
             'subnet': 'subnet1'
         }
         self.cmd('network vnet list-endpoint-services -l westus', checks=[
-            JMESPathCheckV2('length(@)', 1),
+            JMESPathCheckV2('length(@)', 2),
             JMESPathCheckV2('@[0].name', 'Microsoft.Storage')
```

Comment:
unless you know what is the 2nd one, to get the test reliable, you can consider `JMESPathCheckV2("length([?name=='Microsoft.Storage'])", 1)`